spec/index.bs

```diff
@@ -9607,10 +9608,61 @@ dictionary GPUCommandEncoderDescriptor
 
                 1. Let the {{GPUCommandBuffer/[[renderState]]}} of the currently executing
                     {{GPUCommandBuffer}} be a new [=RenderState=].
-                1. Issue: Perform attachment loads/clears.
-            </div>
 
-            Issue: specify the behavior of read-only depth/stencil
+                1. Let {{GPUCommandBuffer/[[renderState]]}}.{{RenderState/[[colorAttachments]]}} be
+                    |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}.
+                1. Let {{GPUCommandBuffer/[[renderState]]}}.{{RenderState/[[depthStencilAttachment]]}} be
+                    |descriptor|.{{GPURenderPassDescriptor/depthStencilAttachment}}.
+
+                1. For each non-`null` |colorAttachment| in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}:
+                    1. Let |colorSubresource| be |colorAttachment|.{{GPURenderPassColorAttachment/depthSlice}} of
```

Comment:
depthSlice is optional so it only makes sense to refer to it when it's 3d.

Could potentially try making `[[colorAttachments]]` be a list of subregions instead of the descriptors, then you don't have to handle this twice (once in begin and once in end).

Reply:
Making `[[colorAttachments]]` a list of subregions would lose the `storeOp`s and `resolveTarget`, so it feels a bit easier to continue to reference the whole attachment descriptor rather than build a similar secondary structure just for the sake of avoiding repeating a couple of algorithm steps.